src/main/java/de/adesso/anki/battle/initializers/SimulationInitializer.java

```diff
@@ -48,10 +49,37 @@ private void addVehicles() {
         vehicle.setTrack(3);
         vehicle.setCurrentRoadpiece(world.getRoadmap().getAnchor().reverse());
         vehicle.setPosition(world.getRoadmap().getAnchor().getEntry().reverse());
-        vehicle.setTargetSpeed(500);
+        vehicle.setTargetSpeed(250);
         vehicle.setMineReady(true);
-
+        vehicle.setRocketReady(true);
         world.addBody(vehicle);
+        
+/*        Vehicle vehicle2 = new Vehicle();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)